Assets/Scripts/UI/LevelSelect.cs

```diff
@@ -19,9 +19,9 @@ public class LevelSelectMenu : MonoBehaviour
     
     [SerializeField] private AudioSource canvasAudioSource;
     [SerializeField] private AudioClip buttonClickSound;
+    [SerializeField] private string level1 = "IntroCutscene";
```

Comment:
this is also stored in `MainMenu.cs`. Since we don't have the play button immediately start the game in MainMenu, we should remove `PlayGame` in `MainMenu.cs` and the starting scene field from there as well.